src/Microsoft.Data/StringDataFrameColumn.cs

```diff
@@ -441,6 +448,14 @@ public override DataFrameColumn FillNulls(object value, bool inPlace = false)
             return column;
         }
 
+        protected override DataFrameColumn FillNullsImplementation(object value, bool inPlace)
+        {
+            if (value is string)
+                return FillNulls((string)value, inPlace);
```

Comment:
Instead of doing 2 type checks, you can just do one if you write the code like this:

```C#
            if (value is string valueString)
                return FillNulls(valueString, inPlace);
```